chat/rich-links/src/RichLink/RichLink.styles.ts

```diff
@@ -0,0 +1,57 @@
+import { css } from '@leafygreen-ui/emotion';
+import { Theme } from '@leafygreen-ui/lib';
+import { createUniqueClassName } from '@leafygreen-ui/lib';
+import { palette } from '@leafygreen-ui/palette';
+import { spacing } from '@leafygreen-ui/tokens';
+
+export const richLinkTextClassName = createUniqueClassName('lg-chat-rich-link');
+
+export const baseStyles = css`
```

Comment:
will the height always be 64px?

Reply:
I think it depends on `baseFontSize` but otherwise we aren't currently wrapping text or expanding the height at all